libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -53,18 +63,20 @@ class AP_BattMonitor
 
     // The BattMonitor_State structure is filled in by the backend driver
     struct BattMonitor_State {
-        cells       cell_voltages;      // battery cell voltages in millivolts, 10 cells matches the MAVLink spec
-        float       voltage;            // voltage in volts
-        float       current_amps;       // current in amperes
-        float       consumed_mah;       // total current draw in milliampere.hours since start-up
-        float       consumed_wh;        // total energy consumed in Watt.hours since start-up
-        uint32_t    last_time_micros;   // time when voltage and current was last read
-        uint32_t    low_voltage_start_ms;  // time when voltage dropped below the minimum
-        float       temperature;        // battery temperature in celsius
-        uint32_t    temperature_time;   // timestamp of the last received temperature message
-        float       voltage_resting_estimate; // voltage with sag removed based on current and resistance estimate
-        float       resistance;         // resistance calculated by comparing resting voltage vs in flight voltage
-        bool        healthy;            // battery monitor is communicating correctly
+        cells       cell_voltages;             // battery cell voltages in millivolts, 10 cells matches the MAVLink spec
+        float       voltage;                   // voltage in volts
+        float       current_amps;              // current in amperes
+        float       consumed_mah;              // total current draw in milliamp hours since start-up
+        float       consumed_wh;               // total energy consumed in Wh since start-up
```

Comment:
If you are going to break git history here, please replace `Wh` with `Watt hours`